README.md

```diff
@@ -5,10 +5,11 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio.svg?style=social&label=Follow&style=for-the-badge)](https://twitter.com/strimziio)
 
-This repository list of proposals for the Strimzi project
+This repository list of proposals for the Strimzi project. The template for new proposals can be found [here](./000-template.md)
 
 |  #  | Title                                                                 |
 | :-: |:----------------------------------------------------------------------|
+|  8  | [Strimzi UI](./008-strimzi-ui.md) |
```

Comment:
shouldn't be 6?

Reply:
I assume he saw the other two PRs and thought to follow on after those.
I agree this should be 6 and the numbering of other proposal PRs should change prior to merge if they go in after this one.

Reply:
I saw two other PRs using 6 and 7 - so I went for 8. Happy to change this to 6.